### PR TITLE
TRUNK-4660 "Administration page does not display"

### PR DESCRIPTION
**Pull request by tomaszmueller**
@rkorytkowski https://issues.openmrs.org/browse/TRUNK-4660

update to jetty 8.0.0 solves the problem

### CODE REVIEW

File: webapp/pom.xml [hunk 1]
```diff
@@ -103,29 +103,29 @@
...
-               <systemProperties>
-                   <systemProperty>
-                       <name>OPENMRS_INSTALLATION_SCRIPT</name>
-                       <value>${OPENMRS_INSTALLATION_SCRIPT}</value>
-                   </systemProperty>
-               </systemProperties>
-            </configuration>
+             <groupId>org.mortbay.jetty</groupId>
+             <artifactId>jetty-maven-plugin</artifactId>
+             <version>8.0.0.M0</version>
```
Comment (rkorytkowski) [inline webapp/pom.xml]: http://mvnrepository.com/artifact/org.eclipse.jetty/jetty-maven-plugin says the latest version is 9.2.10.v20150310 (note the groupId has changed as well) Could you please try it instead?

**Comment by tomaszmueller** [comment]
when building openmrs with jetty 9.2.10.v20150310 I get error on the main page:
An error has occurred!
The following error happened somewhere on this page:
javax.servlet.jsp.JspException: javax.servlet.ServletException: File "/WEB-INF/view//portlets/welcome.jsp" not found

(The full error stack trace output is in the source of this page.)

**Comment by rkorytkowski** [comment]
Ok, then please use the latest version of 8.x


-Rafał

**Comment by tomaszmueller** [comment]
@rkorytkowski updated with 8.1.16

**Pull request merged**